packages/libev.rb

```diff
@@ -0,0 +1,21 @@
+require 'package'
+
+class Libev < Package
+  description 'High-performance event loop loosely modelled after libevent'
+  homepage 'http://software.schmorp.de/pkg/libev.html'
+  version '4.33'
+  compatibility 'all'
+  source_url 'http://dist.schmorp.de/libev/libev-4.33.tar.gz'
+  source_sha256 '507eb7b8d1015fbec5b935f34ebed15bf346bed04a11ab82b8eee848c4205aea'
+
+  depends_on 'binutils'
+  
+  def self.build
+    system "./configure #{CREW_OPTIONS} --libdir=#{CREW_LIB_PREFIX}"
```

Comment:
`--libdir=#{CREW_LIB_PREFIX}` should already be included in `#{CREW_OPTIONS}`.  See `crew const CREW_OPTIONS`.